featurebyte/service/offline_store_feature_table_manager.py

```diff
@@ -55,9 +55,18 @@ async def handle_online_enabled_feature(self, feature: FeatureModel) -> None:
         ):
             feature_entities.append(entity_model)
 
-        offline_ingest_graphs = feature.extract_offline_store_ingest_query_graphs(
-            {entity_model.id: entity_model.serving_names[0] for entity_model in feature_entities}
-        )
+        # FIXME: cleanup this logic once integrate this with feast feature store
+        if feature.offline_store_info is None:
+            feature.initialize_offline_store_info(
```

Comment:
Is this something that should be done when saving features, but we don't do it now on purpose before the schema is finalised?

Reply:
Yes, we don't do it now on purpose before schema is finalised. Not sure whether we should do it when saving features or only when it is online enable.